server/graphql/v2/object/Host.js

```diff
@@ -1091,23 +1091,36 @@ export const GraphQLHost = new GraphQLObjectType({
 
           const pageQuery = `
                 WITH hostadmins AS (
-                  SELECT "MemberCollectiveId" FROM "Members" WHERE "CollectiveId" = :hostid AND "deletedAt" IS NULL AND role = 'ADMIN'
-                ), orgs AS (
-                  SELECT c.id, c.slug,ARRAY_AGG(DISTINCT m."MemberCollectiveId") as "admins", ARRAY_AGG(DISTINCT t."HostCollectiveId") as hosts
+                  SELECT m."MemberCollectiveId", m."CreatedByUserId" as "UserId"
+                  FROM "Members" m
+                  INNER JOIN "Users" u ON m."MemberCollectiveId" = u."CollectiveId"
+                  WHERE m."CollectiveId" = :hostid AND m."deletedAt" IS NULL AND m.role = 'ADMIN'
+                  ), orgs AS (
+                  SELECT c.id, c.slug,ARRAY_AGG(DISTINCT m."MemberCollectiveId") as "admins", ARRAY_AGG(DISTINCT t."HostCollectiveId") as hosts, c."CreatedByUserId"
                   FROM "Collectives" c
                   LEFT JOIN "Members" m ON c.id = m."CollectiveId" AND m."deletedAt" IS NULL AND m.role = 'ADMIN'
                   LEFT JOIN "Transactions" t ON c.id = t."FromCollectiveId" AND t."deletedAt" IS NULL
                   WHERE c."deletedAt" IS NULL
                     AND c.type = 'ORGANIZATION'
                     AND c."HostCollectiveId" IS NULL
                   GROUP BY c.id
-                )
+                  )
+
                 SELECT c.*
-                FROM "orgs"
-                INNER JOIN "Collectives" c ON c.id = "orgs".id
+                FROM "orgs" o
+                INNER JOIN "Collectives" c ON c.id = o.id
                 WHERE
-                  "admins" <@ ARRAY(SELECT "MemberCollectiveId" FROM hostadmins)
-                  AND "hosts" = ARRAY[:hostid]
+                  (
+                    o."admins" <@ ARRAY(SELECT "MemberCollectiveId" FROM hostadmins)
+                      OR (
+                        o."CreatedByUserId" IN (
+                        SELECT "UserId"
+                        FROM hostadmins
+                        )
+                        AND o."admins" = ARRAY[null]::INTEGER[]
```

Comment:
I'm not convinced by this `ARRAY[null]::INTEGER[]`, organizations created via added funds do have an admin, it's just that it's an invited account (that often doesn't get confirmed)

Reply:
Yes. I was working with the idea that I didn't want to block Organizations that were never activated from becoming Vendors. (Assuming the invited admin never had the intention to join).